### ISSUE TITLE
Please compile with JDK 1.8 instead of JDK 11 to preserve compatibility with 1.8

### ISSUE THREAD

**Issue by archiecobbs** [docbook/xslTNG#143]
The distribution Java class files are not compatible with Java 1.8:
```shell
$ java -jar /usr/share/docbook-xsltNG/libs/docbook-xslTNG-1.7.0-11.jar 
Error: A JNI error has occurred, please check your installation and try again
Exception in thread "main" java.lang.UnsupportedClassVersionError: org/docbook/xsltng/Main has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:757)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:473)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:74)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:369)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:363)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:362)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:419)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:352)
	at sun.launcher.LauncherHelper.checkAndLoadMain(LauncherHelper.java:601)
```

Presumably this is because the distribution is being built with JDK 11.

However, this is not strictly necessary. All of the JAR dependencies included in the are 1.8 compatible, so I think it should be possible to compile xsltNG with JDK 1.8, or at least with JDK 11 but with `-source` and `-target` of 1.8.

That would help with the many systems out there that are still running 1.8. 

Thanks!

**Comment by ndw** [comment]
Thanks for the report. I don't think I had any compelling reason to move to 11, so that's probably just an accident on my part. I'll push another release later this week.

**Comment by archiecobbs** [comment]
Awesome, thanks!

**Comment by ndw** [comment]
I believe this is fixed in 1.7.1 just released a few minutes ago.
